src/lib/layouts/workflow-header.svelte

```diff
@@ -136,6 +129,15 @@
           {workflow?.historyEvents}
         </Badge>
       </Tab>
+      <Tab
+        label={translate('workflows.relationships')}
+        id="relationships-tab"
+        href={routeForRelationships(routeParameters)}
+        active={pathMatches(
+          $page.url.pathname,
+          routeForRelationships(routeParameters),
+        )}
+      />
```

Comment:
I removed the count since we make the query when you go to the tab. And also... would we want to show the full count of relationship (parent/children/grandchildre/cousins) or just children??